application/src/Controller/BackOfficeController.php

```diff
@@ -33,7 +34,13 @@ public function backOfficeMeetingCreate(string $serverID, Request $request): Xml
     {
         $meeting = new Meeting();
         $meeting->setServerID($serverID);
-        $meeting->setMeetingId($request->query->get('meetingID'));
+        // Deal first with Recording that are part of a breakroom meeting.
```

Comment:
Hello Everyone,
I'm not native English so I could be wrong but _breakroom_ and _breakout room_ look different to me.

The code seems to have more _breakout room_ refs so it looks like a typo to me.

HTH,
Matteo